app/models/concerns/image_file.rb

```diff
@@ -79,7 +79,7 @@ def url
   end
 
   def href
-    status_complete? ? url : original_url
+    (status_complete? || status_invalid?) ? url : original_url
```

Comment:
Checking for understanding: This is making sure that if either `status_complete` or `status_invalid` are valid, it goes on to hit `url`, checking first for the `f.prxu.org` as the source, but if not, will fall back on the `original_url`.

Reply:
You got it -

If the status is `"complete"` or `"invalid"`, we know that Porter succeeded, and `"https://f.prxu.org/..."` (the string returned by `url`) will return the file.

If the status is something else (error, processing, started, etc) ... Porter hasn't managed to copy the file to `https://f.prxu.org/...` for some reason.  And trying to GET it will 404.  So we should use the `original_url`.